lib/repl.js

```diff
@@ -287,7 +287,8 @@ function REPLServer(prompt,
     const top = replMap.get(self);
     const pstrace = Error.prepareStackTrace;
     Error.prepareStackTrace = prepareStackTrace(pstrace);
-    internalUtil.decorateErrorStack(e);
+    if (typeof e === 'object')
```

Comment:
to protect against the odd random non-object being emitted on the domain error event. If `e` is not an object, it will trigger the `CHECK`